compiler+runtime/src/cpp/jank/runtime/core.cpp

```diff
@@ -556,6 +556,12 @@ namespace jank::runtime
     return o->type == object_type::volatile_;
   }
 
+  object_ptr vswap(object_ptr const v, object_ptr const fn)
+  {
+    auto const v_obj(expect_object<obj::volatile_>(v));
```

Comment:
I know you're just following what's there, but this makes me realize that `vswap` (both overloads) and `vreset` need `try_object` instead of `expect_object`. Otherwise, giving it anything else will not throw. It'll just be UB.